_assignments/mp4.md

```diff
@@ -65,15 +65,17 @@ rubric:
   -
     name: Framerate/Comfort
     points: 20
-    description: Framerate rarely drops below 60 fps, game sound is not obnoxious, and plane acceleration, deceleration, and rotations are comfortable.
+    description: Framerate rarely drops below 60 fps, game sound is not obnoxious, and plane acceleration, deceleration, and rotations are comfortable. Game conforms to Oculus Best Practices Guide
   -
     name: Interactive tutorial
     points: 20 EC
     description: The tutorial provided to the player is interactive
-  -
 
 points: 200
 ---
+## Overview
+Welcome to MP4! In this MP, you are going to build a flight simulator in Unity. You are going to construct a start menu and tutorial, write script to support your plane, make a UI, and optimize the game to fulfill requirements from the [Oculus Best Practices Guide](https://developer.oculus.com/design/latest/concepts/book-bp/)!Much of the spec for this MP is vague. This is intentional, as we want you to be in the practice of finding creative solutions to problems. Note: This is an assignment with a lot of details, so please start early.
+**PLEASE REGULARLY SAVE YOUR WORK IN THIS MP, AS UNITY MIGHT CRASH AT ANY TIME!**

```

Comment:
Add an empty line between line 77 and 78 so that line 78's text shows in new line